$NetBSD: patch-configure,v 1.5 2020/02/05 20:18:27 tnn Exp $

Use USE_AMD64_IOPL (actually x86_64_iopl) on NetBSD/amd64.
(c.f. patch-hw_xfree86_os-support_bsd_i386.c)

for netbsd, force pass pthread configure checks (pthread not needed)

--- configure.orig	2020-01-13 22:57:18.000000000 +0000
+++ configure
@@ -23146,7 +23146,7 @@ $as_echo "#define USE_ALPHA_PIO 1" >>con
 	esac
 	GLX_ARCH_DEFINES="-D__GLX_ALIGN64 -mieee"
 	;;
-  arm*)
+  *arm*|aarch64)
 	ARM_VIDEO=yes
 	DEFAULT_INT10="stub"
 	;;
@@ -23188,7 +23188,7 @@ $as_echo "#define USE_DEV_IO 1" >>confde
 $as_echo "#define USE_DEV_IO 1" >>confdefs.h
  ;;
 		*netbsd*)
-$as_echo "#define USE_I386_IOPL 1" >>confdefs.h
+$as_echo "#define USE_AMD64_IOPL 1" >>confdefs.h
 
 				SYS_LIBS=-lx86_64
 				;;
@@ -25010,7 +25010,8 @@ case ${host_os} in
 	netbsd*)
 	# use libc stubs, don't link against libpthread, to allow
 	# dynamic loading
-	ax_pthread_flags=""
+	ax_pthread_flags="none"
+	ax_pthread_ok="yes"
 	;;
 esac
 
